components/places/src/places.udl

```diff
@@ -93,6 +143,38 @@ dictionary HistoryHighlight {
     string? preview_image_url;
 };
 
+dictionary HistoryVisitInfo {
+    string url;
```

Comment:
Can we use `Url`s here and below? APIs like `getVisited()` which intentionally ignore invalid URLs can't, but I don't think most of these are in this situation.